### ISSUE TITLE
Site styling is broken over HTTPS.

### ISSUE THREAD

**Issue by ivan-kolmychek** [CodeHubApp/CodeHub#351]
I am not entirely sure I open the issue in the correct repo, but I haven't found any repos with issues for `https://codehub-app.com/` itself.

**TL;DR:** 

Can you replace your bootstrap and font-awesome links to HTTPS versions? It's `https://netdna.bootstrapcdn.com/bootstrap/3.0.0/css/bootstrap.min.css` and `https://netdna.bootstrapcdn.com/font-awesome/4.0.0/css/font-awesome.min.css` accordingly.

**Long version:**

When accessing the site over HTTPS, the styling breaks a bit:

![screenshot_20171102_083942](https://user-images.githubusercontent.com/5011577/32314727-7e4fae0e-bfa9-11e7-9abd-dcf262d6c45f.png)
![screenshot_20171102_083951](https://user-images.githubusercontent.com/5011577/32314728-7e7227ea-bfa9-11e7-97e9-4ad0f7138ee9.png)

From what I see in browser console, it's because of `http://netdna.bootstrapcdn.com/bootstrap/3.0.0/css/bootstrap.min.css` and `http://netdna.bootstrapcdn.com/font-awesome/4.0.0/css/font-awesome.min.css` being loaded over HTTP - they are being blocked on HTTPS version then because they're a [mixed content](https://developer.mozilla.org/en-US/docs/Web/Security/Mixed_content).

I've just checked can you load them over HTTPS and it seems you can, and making them use it by default should fix this issue without breaking anything - loading of HTTPS content from HTTP page is OK.

Hope it helps. :)

**Comment by thedillonb** [comment]
Wow. Good catch. I just assumed since GitHub doesn't seem to offer SSL on custom domains that nobody would access it over HTTPS. I've made the changes. Thanks for pointing this out!
